### ISSUE TITLE
Screen Reader: Settings-Profile: The Search button has no role announced

### ISSUE THREAD

**Issue by nbhardwaj-del** [Expensify/App#74888]
## Action Performed
Prerequisites:
- The user is signed in

1. Using JAWS + Chrome, open the site https://new.expensify.com/settings/profile
2. Press Tab key to focus the Search button at the top right corner.
3. Note how the control is announced.

## Expected Result
The Search button should be announced with a button role.

## Actual Result
When users navigate to the search button element, it is announced without the role. Screen reader only announces it as 'Search'. Users will not understand this is an actionable element

HTML:
<div aria-label="Search" data-tag="pressable" tabindex="0" class="css-g5y9jx r-1otgn73 r-1loqt21 r-18u37iz r-1awozwy r-eu3ka r-1777fci r-1aockid" data-testid="searchButton" data-component="GenericPressable" data-element="Pressable" data-source-file="BaseGenericPressable.tsx"><div class="css-g5y9jx" data-testid=""><svg xmlns="[http://www.w3.org/2000/svg"](http://www.w3.org/2000/svg%22) xml:space="preserve" viewBox="0 0 20 20" width="20" height="20" hovered="false" pressed="false" fill="#A2A9A3"><path d="M8 1c3.9 0 7 3.1 7 7 0 1.4-.4 2.7-1.1 3.8l5.2 5.2c.6.6.6 1.5 0 2.1s-1.5.6-2.1 0l-5.2-5.2C10.7 14.6 9.4 15 8 15c-3.9 0-7-3.1-7-7s3.1-7 7-7m0 3c2.2 0 4 1.8 4 4s-1.8 4-4 4-4-1.8-4-4 1.8-4 4-4" style="fill-rule: evenodd; clip-rule: evenodd;"></path></svg></div></div>

## Area issue was found
Settings–Profile

## Failed Checkpoint
WCAG 4.1.2 Name, Role, Value
https://www.w3.org/WAI/WCAG22/Understanding/name-role-value

## User Impact
Providing role, state, and value information on all user interface components enables compatibility with assistive technology, such as screen readers, screen magnifiers, and speech recognition software, used by people with disabilities.

## Suggested Resolution
Implement the control as a native button.
https://www.w3.org/WAI/ARIA/apg/patterns/button/

## Environment(s)
Windows Windows 10 - Chrome

## Screenshots/Videos
https://github.com/user-attachments/assets/4d35d17b-33e8-47e1-b17d-834590719133

<img width="1921" height="1003" alt="Image" src="https://github.com/user-attachments/assets/9d8f1db0-1ba6-43b1-9767-c87fc5a12ba2" />

**Version Number:** NA
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** NA
**Logs:** https://stackoverflow.com/c/expensify/questions/4856 NA
**Notes/Photos/Videos:** Any additional supporting documentation Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36605/community-issues/6985956
**Issue reported by:** NA
**Slack conversation:** NA

**Comment by mallenexpensify** [comment]
Accessibility, unassigning, will be addressed in bulk once all issues are created.
